<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="zoomInBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>move</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>366</width>
       <height>296</height>
      </rect>
     </property>
     <widget class="QLabel" name="imageLabelIn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaOut">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>379</width>
       <height>309</height>
      </rect>
     </property>
     <widget class="QLabel" name="imageLabelOut">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboFilter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Median filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Avarage filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low-pass filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss filter</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select noise filter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set kernel size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSize">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3 x 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 x 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 x 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 x 9</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="applyFilterBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>98</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImgBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>98</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save otput</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
